lib/index.js

```diff
@@ -105,6 +105,12 @@ exports.parseCOinSTitle = BBPromise.method(function(title){
 		}
 		if (key.length === 2){ // Split key e.g. rft.date
 			if (key[0] !== 'rft') {return;} // Invalid hierarchical key
+			// Keys that may have multiple values - return in list format
+			if (key[1] === 'au' || key[1] === 'isbn' || key[1] === 'issn' || key[1] === 'eissn' || key[1] === 'aucorp') {
+				if (!rft[key[1]]) {rft[key[1]] = [];}
+				rft.au.push(value);
```

Comment:
Pushing everything to `au` instead of `rft[key[1]]` looks suspicious, looks like a bug.